docusaurus/docs/e2e-test-a-plugin/migrate-from-grafana-e2e.md

```diff
@@ -0,0 +1,172 @@
+        // @grafana/plugin-e2e writes the auth state to this file,
+        // the path should not be modified
+        storageState: 'playwright/.auth/admin.json',
+      },
+      dependencies: ['auth'],
+    }
+  ],
+});
+```
+
+The authenticated state is stored on disk with the following file name pattern: `<plugin-root>/playwright/.auth/<username>.json`.
+
+To prevent these files from being version controlled, you can add the following line to your `.gitignore` file:
+
+```shell title=".gitignore"
+/playwright/.auth/
+```
+
+The `@grafana/plugin-e2e` package also exports a type named `PluginOptions` that you can use to extend the Playwright configuration with the `@grafana/plugin-e2e` specific options.
+
+```ts title="playwright.config.ts"
+import { defineConfig } from '@playwright/test';
+import type { PluginOptions } from '@grafana/plugin-e2e';
+
+export default defineConfig<PluginOptions>({
+    ...
+});
+```
+
+You're now ready to start writing tests. The [Get started](./get-started.md) guide to learn how to run tests locally.
+
+## Migrating tests
+
+There's no tooling in place for automatically migrating existing `@grafana/e2e` based Cypress tests to `@grafana/plugin-e2e` based Playwright tests. This means you would have to convert your tests one by one or replace them with a new set of tests based on Playwright. You may checkout the following resources to get inspiration on how the playwright tests should be written:
+
+- [How to test data source plugins](./test-a-data-source-plugin/index.md)
+<!-- - [How to test panel plugins](./test-a-panel-plugin.md) -->
+- [App plugin with example tests](https://github.com/grafana/grafana-plugin-examples/tree/main/examples/app-basic/tests)
+- [Panel plugin with example tests](https://github.com/grafana/grafana-plugin-examples/tree/main/examples/panel-datalinks/tests)
+- [Best practices around test isolation](./setup-resources.md#test-isolation)
+- `@grafana/plugin-e2e` exposes Grafana end-to-end selectors via the `Selectors` fixture, so you no longer need to use selectors exposed via the `@grafana/e2e-selectors` package. Read more about end-to-end selectors in the [Selecting UI elements](./selecting-ui-elements.md) guide.
+
+## Running tests in CI
+
+To run Playwright tests targeting multiple versions of Grafana in CI, use one of the example workflows in the [CI](./ci.md) guide.
+
+:::note
+
+Note that Grafana does not offer any supported way to run end-to-end tests targeting multiple versions of Grafana in other CI platforms such as Drone or CircleCI. But you can easily configure your CI to replicate what the referenced Github Action is doing as there is nothing specific that we do that can not be done in other CI systems.
+
+:::
+
+## Uninstalling Cypress and @grafana/e2e
+
+Although we recommend moving from `@grafana/e2e` to `@grafana/plugin-e2e` in a timely manner, there's nothing preventing you from having the two side by side during a transitional phase.
```

Comment:
We could warn here about issues targeting versions greater than 11 using cypress

Reply:
I don't think it's necessary. We've already made it clear at the top of the page that the package won't be supported after G11.

Reply:
I think theres a difference between not being supported and not working entirely, people might just expect that no new features would be added and fixes not applied. But not a blocker by any sense